saplings/contracts/public/manifest.json

```diff
@@ -6,4 +6,3 @@
   "theme_color": "#000000",
   "background_color": "#ffffff"
 }
-  
```

Comment:
No newline at end of file

Reply:
@dplumb94 I think you might be looking at an outdated version of the file.